### ISSUE TITLE
No default keybindings

### ISSUE THREAD

**Issue by seanabraham** [brandoncc/vscode-seeing-is-believing#2]
I think it would be nice to add some default keybindings. These are what I'm using for now:

```json
[
    {
        "key": "ctrl+cmd+m",
        "command": "seeing-is-believing.toggle-marks",
        "when": "editorTextFocus"
    },
    {
        "key": "ctrl+cmd+r",
        "command": "seeing-is-believing.run",
        "when": "editorTextFocus"
    }
]
```

**Comment by brandoncc** [comment]
I agree that it would be nice to include default bindings. I'm always afraid of causing conflicts for the users though and tend to leave them to the users to create. I'll give this some more thought, especially since cmd is Mac specific so I'm not sure if it would automatically be translated to `win` by VSC or not. I'll have to try it and see :+1:
